frontend/ui/data/css_tweaks.lua

```diff
@@ -650,12 +650,16 @@ This is just an example, that will need to be adapted into a user style tweak.]]
                     title = _("In-page FB2 footnotes"),
                     description = _([[
 Show FB2 footnote text at the bottom of pages that contain links to them.]]),
+                    -- Restrict this to FB2 documents, even if we won't probably
+                    -- match in any other kind of document
                     css = [[
 body[name="notes"] section {
+-cr-only-if: fb2-document;
```

Comment:
This rather… distracting indentation is on purpose to make it stand out?

Reply:
Yes. I used that kind of layout to express more clearly what the "only if" affects, as in:
https://github.com/koreader/koreader/blob/201d3d8ad1c9e6432283439e3c14abfd9569bc2f/frontend/ui/wikipedia.lua#L1015-L1039
I can add another level of indentation if you prefer, we have enough width to show them non wrapped:
![image](https://user-images.githubusercontent.com/24273478/87934401-41b90500-ca8f-11ea-9c44-b16105134de0.png)
=>
![image](https://user-images.githubusercontent.com/24273478/87934480-6e6d1c80-ca8f-11ea-9145-e14a76e6f235.png)

Reply:
I think that would be nicer to look at (like in the Wikipedia one). :+1: